ehrapy/api/data/datasets.py

```diff
@@ -1,23 +1,27 @@
 from anndata import AnnData
 
+from ehrapy.api.encode import Encoder
+from ehrapy.api.io import DataReader
 
-def mimic_2() -> AnnData:
-    """Loads the mimic-ii dataset.
 
-    The dataset is available here: https://physionet.org/content/mimic2-iaccd/1.0/
+class Datasets:
+    @staticmethod
+    def mimic_2(encode: bool = False) -> AnnData:
+        adata = DataReader.read(
+            filename="ehrapy_mimic2.csv",
+            backup_url="https://www.physionet.org/files/mimic2-iaccd/1.0/full_cohort_data.csv?download",
+            suppress_warnings=True,
+        )
+        if encode:
+            return Encoder.encode(adata, autodetect=True)
```

Comment:
Autodetect would now just label encode everything, right? That's not how it should be.

There are some columns that should def be one-hot encoded.

Reply:
Mh, I changed this because of the encoding for the caching file. 
I mean, one could just pass an _encoding_mode_ parameter, but it's not possible to apply an encoding mode to specific columns as it is the case without autodetect.